examples/tutorials/05_ordinary_torch_models.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p>Training might be continued as the loss is not on a plateau yet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>